src/components/FavoriteItem/FavoriteItem.js

```diff
@@ -0,0 +1,148 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+
+import classNames from 'classnames'
+import { ButtonWithConfirm } from 'components'
+import { JText, JCard, JIcon, JTooltip } from 'components/base'
+
+type Props = {
+  editItem: () => void,
+  deleteItem: () => void,
+  address: Address,
+  +name: string,
+  +symbol: string,
+}
+
+type StateProps = {|
+  isToggled: boolean,
+  isHoveredSend: boolean,
+  isHoveredEdit: boolean,
+  isHoveredTrash: boolean,
+|}
+
+class FavoriteItem extends PureComponent<Props, StateProps> {
+  static defaultProps = {
+    isCustom: false,
+    isActive: false,
+  }
+
+  constructor(props: Props) {
+    super(props)
+
+    this.state = {
+      isToggled: false,
+      isHoveredSend: false,
+      isHoveredEdit: false,
+      isHoveredTrash: false,
+    }
+  }
+
+  onHoverSend = () => this.setState({ isHoveredSend: !this.state.isHoveredSend })
+
+  onHoverEdit = () => this.setState({ isHoveredEdit: !this.state.isHoveredEdit })
+
+  onHoverTrash = (isHoveredTrash: boolean) => () => this.setState({ isHoveredTrash })
+
```

Comment:
all onHover handlers should be the same as `onHoverTrash`
or even better make one handler for changing `hoveringItem: 'send' | 'edit' | 'delete'` state prop, as you wish